test/Microsoft.ApplicationInsights.AspNetCore.Tests/Extensions/ApplicationInsightsExtensionsTests.cs

```diff
@@ -585,6 +585,17 @@ public static void AddsQuickPulseProcessorToTheConfigurationByDefault()
                 Assert.Equal(1, qpProcessorCount);
             }
 
+            [Fact]
+            public static void AddsAutoCollectedMetricsExtractorProcessorToTheConfigurationByDefault()
```

Comment:
Do we need any test actually confirming it's operational after it has been added? Or we're OK it's covered in the other repos?

Reply:
This repo simply does the configuratin i.e ensure the processor is added. Its coming from base sdk and tested there. (same is followed for pretty much everything in this repo like DepCollector/perfCollector. etc)